dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/expand/CuringGlobalParams.java

```diff
@@ -206,6 +206,28 @@ public Map<String, Property> paramParsingPreparation(@NonNull TaskInstance taskI
         return globalParams;
     }
 
+    /**
+     * build all built-in parameters
+     * @param taskInstance
+     * @param timeZone
+     */
+    private Map<String, String> setBuildInParamsMap(@NonNull TaskInstance taskInstance,
```

Comment:
In my opinion you should change the function name from setBuildInParamsMap to setBuiltInParamsMap.

Reply:
make sense